issue_template.md

```diff
@@ -0,0 +1,42 @@
+<!--
+ 1. IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MAY CLOSE YOUR ISSUE WITHOUT INVESTIGATION
+ 2. SEARCH EXISTING ISSUES FOR AN ANSWER: https://goo.gl/WiaCcG
+ 3. See our Common Issues documentation: https://goo.gl/68ntg2
+-->
+
+**Description:**
+<!-- (write below this line) -->
+
+(your description here)
+
+**Environment**
+<!-- Example:
+1. What version of the iOS SDK are you using?
+2. How did you add the SDK to your project (eg. cocoapods)
+ -->
+
+
+
+**Steps to Reproduce Issue:**
+<!--
+  Example:
+
+  1. Visit https://site.com/page.
```

Comment:
Should be something like open app instead of visit site...

Reply:
Of course 🤦‍♂️